tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Grpc/GrpcCommon.cs

```diff
@@ -0,0 +1,119 @@
+﻿// <copyright file="GrpcCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Grpc
+{
+    internal static class GrpcCommon
+    {
+        public const string OperationName = "grpc.request";
```

Comment:
Are we using the same operation name for server and client spans? If so, we should probably not do that. I will cause issues if a trace contains both types of spans. (backend doesn't distinguish them by `SpanKind` yet.)

Reply:
The [RFC](https://github.com/DataDog/architecture/blob/master/rfcs/apm/integrations/integrations/grpc/rfc.md#normalized-behaviour) specifically requires using the same name though, and AFAIK all other tracers do&hellip; 😬

Reply:
Looks like this may be ok for now (it _can_ be an issue, but it's low risk). The upcoming project to unify span names and values may resolve this. Thanks!